modular_sojourn/general_things/perks.dm

```diff
@@ -0,0 +1,96 @@
+
+#define PERK_HUMAN_ARMOUR_BOOST /datum/perk/job/natural_armour
+/datum/perk/job/natural_armour
```

Comment:
This ADDS 20% more brute and burn damage (10% o2)
flip the assign to be -=
flip the remove to be +=

Reply:
done